web-www/src/sections/nct/LedgerSection.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
+import SanityImage from 'gatsby-plugin-sanity-image';
+import BackgroundImage from 'gatsby-background-image';
+import { FluidObject } from 'gatsby-image';
+import { Box } from '@mui/material';
+
+import Section from 'web-components/lib/components/section';
+import Title from 'web-components/lib/components/title';
+import { BlockContent } from 'web-components/src/components/block-content';
+import { Label } from 'web-components/lib/components/label';
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+
+import type { NctLedgerSectionQuery } from '../../generated/graphql';
+
+const query = graphql`
+  query nctLedgerSection {
+    bg: file(relativePath: { eq: "topo-bg-top.png" }) {
+      childImageSharp {
+        fluid(quality: 90) {
+          ...GatsbyImageSharpFluid_withWebp
+        }
+      }
+    }
+    sanityNctPage {
+      ledgerSection {
+        icon {
+          ...ImageWithPreview
+        }
+        title
+        subtitle
+        _rawBody
+        button {
+          ...buttonFields
+        }
+      }
+    }
+  }
+`;
+
+export const LedgerSection = (): JSX.Element => {
+  const { bg, sanityNctPage } = useStaticQuery<NctLedgerSectionQuery>(query);
+  const data = sanityNctPage?.ledgerSection;
+
+  return (
+    <Box
+      sx={{
+        borderTop: 1,
+        borderBottom: 1,
+        borderColor: 'grey.100',
+      }}
+    >
+      <BackgroundImage fluid={bg?.childImageSharp?.fluid as FluidObject}>
+        <Section sx={{ py: [20, 30] }}>
+          <Box
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              flexDirection: 'column',
+              gap: 5,
+              maxWidth: 760,
+              m: '0 auto',
+            }}
+          >
+            <SanityImage {...(data?.icon as any)} alt="Regen icon" />
+            <Title variant="h2" sx={{ mt: [4, 8], textAlign: 'center' }}>
+              {data?.title}
+            </Title>
+            <Label
+              sx={{
+                color: 'info.main',
+                fontSize: [14, 18],
+                textAlign: 'center',
+              }}
+            >
+              {data?.subtitle}
+            </Label>
+            <BlockContent
+              noYMargin
+              content={data?._rawBody}
+              sxWrap={{ '& p': { fontSize: [18, 22], textAlign: 'center' } }}
+            />
+            <OutlinedButton
+              sx={{ px: [8, 12], mt: [4, 8] }}
+              href={data?.button?.buttonLink?.buttonHref || ''}
+              target={data?.button?.buttonBlankTarget ? '_blank' : '_self'}
```

Comment:
I wasn't passing `target` before - this should fix (also added elsewhere)

Reply:
It'd be great to have that coded at the button component level so we don't have to repeat this logic every time, could you create a code-hygiene issue to track that?
we have some kind of util function already `onBtnClick` but it's on web-registry for now so should be adapted to work for both CRA and Gastby.

Reply:
@blushi sorry I missed this comment before merging! 

Looking at the current helpers, they aren't using react router so the site will reload upon clicking. Created a story and added some thoughts [here](https://app.zenhub.com/workspaces/regen-registry-5f8998bec8958d000f4609e2/issues/regen-network/regen-registry/881)